src/list/actions.js

```diff
@@ -331,3 +344,47 @@ export function openFAQ() {
     type: OPEN_FAQ,
   };
 }
+
+export function openSyncPrefs() {
+  window.open(SYNC_PREFS_URL, "_blank");
```

Comment:
as noted in IRL and chats, this needs to be an API call on `browser.experiments.sync` ... can be deferred to #92 / #61